packages/db/index.js

```diff
@@ -177,6 +157,38 @@ export class DBClient {
     return data.length ? data[0].value : undefined
   }
 
+  /**
+   * Returns all the active (non-deleted) user tags for a user id.
+   *
+   * @param {number} userId
+   * @returns {Promise<string[] | undefined>}
```

Comment:
Should this be `@returns {Promise<object[]>}` or `@returns {Promise<{tag: string, value: string}[]>`?